Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -207,6 +212,66 @@ Describe "New-GcsObject" {
         $obj.Metadata["Content-Type"] | Should Be "image/jpeg"
         Remove-GcsObject $obj
     }
+
+    It "should upload an empty folder" {
+        $TestFolder = Join-Path $TestDrive "TestFolder"
```

Comment:
It should be camelCased. My bad. I'll fix it. $TestDrive is a variable created automatically in a single Describe or Context block of Pester (https://github.com/pester/Pester/wiki/TestDrive).